saffron/e2e-test.sh

```diff
@@ -11,6 +11,7 @@ SRS_ARG=""
 if [ $# -eq 2 ]; then
    SRS_ARG="--srs-filepath $2"
 fi
+COMMITMENT_FILE="${INPUT_FILE%.*}_commitment.bin"
```

Comment:
Same comment that [here](https://github.com/o1-labs/proof-systems/pull/3004#discussion_r1944461370).